### PR TITLE
docs: add TopicOverviewCard

### LINKED ISSUE #162: Build Doku Component - Topic Overview Card

### PR DESCRIPTION
**Pull request by larsrickert**
closes #162

## Description

Add a topic overview card component for our VitePress docs. Can be used with:
```html
<div class="topic-overview">
  <TopicOverviewCard title="Card title" subtitle="Subtitle..." image-src="https://picsum.photos/128" href="/some/page" />
  <TopicOverviewCard title="Card title" subtitle="Subtitle..." image-src="https://picsum.photos/128" href="/some/page"/>
  <TopicOverviewCard title="Card title" subtitle="Subtitle..." image-src="https://picsum.photos/128" href="/some/page" />
  <TopicOverviewCard title="Card title" subtitle="Subtitle..." image-src="https://picsum.photos/128" href="/some/page"/>
  <TopicOverviewCard title="Card title" subtitle="Subtitle..." image-src="https://picsum.photos/128" href="/some/page"/>
</div>
```
image

## Checklist

- [x] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
- [x] All changes are documented in the documentation app (folder `apps/docs`)
- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

**Review by jannick-ux — DISMISSED** [review]

**Review by jannick-ux — APPROVED** [review]

File: apps/docs/src/.vitepress/theme/theme.scss [hunk 1]
```diff
@@ -19,3 +19,9 @@
     }
   }
 }
+
+.topic-overview {
```
Comment (BoppLi) [inline apps/docs/src/.vitepress/theme/theme.scss:23]: this is not used yet, intentionally?

File: apps/docs/src/.vitepress/components/TopicOverviewCard.vue [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<script lang="ts" setup>
+const props = defineProps<{
+  title: string;
+  subtitle: string;
+  image: string;
```
Comment (BoppLi) [inline apps/docs/src/.vitepress/components/TopicOverviewCard.vue:5]: just a suggestion, you decide :)
```suggestion
  imageSrc: string;
```

File: apps/docs/src/.vitepress/components/TopicOverviewCard.vue [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  href: string;
+}>();
+</script>
+
+<template>
+  <a class="card" tabindex="0" :href="props.href">
```
Comment (BoppLi) [inline apps/docs/src/.vitepress/components/TopicOverviewCard.vue:11]:
```suggestion
  <a class="topic" tabindex="0" :href="props.href">
```

yes, same as last time with the component card: both are cards, but from the semantics, they represent different things ^^

**Review by BoppLi — CHANGES_REQUESTED** [review]

**Comment by larsrickert** [comment]
Suggestion: Once all images are final we should convert them to webp to reduce image size

File: apps/docs/src/.vitepress/theme/theme.scss (hunk 1 above)
Comment (larsrickert) [reply apps/docs/src/.vitepress/theme/theme.scss:23]: Yes the components will be used by Jannick in his PR. This is just the technical setup for it

**Review by BoppLi — APPROVED** [review]

**Pull request merged**